lib/custom_widgets/settings_card.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+
+class SettingsCard extends StatelessWidget {
+  final settingName;
+  final settingDescription;
+  final settingIcon;
+
+  const SettingsCard({
+    super.key,
+    required this.settingName,
+    required this.settingDescription,
+    required this.settingIcon,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+        color: const Color.fromRGBO(217, 217, 217, 1),
+        margin: const EdgeInsets.only(left: 20.0, right: 10.0, top: 20.0),
+        child: SizedBox(
+            width: 171,
+            height: 114,
+            child: Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Container(
+                      decoration: const BoxDecoration(
+                          shape: BoxShape.circle, color: Colors.white),
+                      child: Padding(
+                          padding: const EdgeInsets.all(8.0),
+                          child: settingIcon),
+                    ),
+                    const SizedBox(height: 8),
+                    Text(settingName,
+                        style: const TextStyle(
+                            color: Colors.black,
+                            fontSize: 18.0,
+                            fontFamily: 'SF Pro Text',
+                            fontWeight: FontWeight.bold)),
+                    const SizedBox(height: 4),
+                    Text(settingDescription,
+                        style: const TextStyle(
+                          color: Colors.black,
+                          fontSize: 12.0,
+                          fontFamily: 'SF Pro Text',
```

Comment:
I'd suggest you to use a macro for the font (and replace it where necessary, also in the other two widgets)

I apologize if you do not know what a macro is, I mean something like a global variable. 

**Let's talk about whether it is better to create a new file for the macros**